types/stripe/index.d.ts

```diff
@@ -1469,6 +1470,38 @@ declare namespace Stripe {
             source?: sources.ISourceCreationOptionsExtended;
         }
 
+        interface ICustomerListOptions {
```

Comment:
Sure, let me update it.

Reply:
@squirly Actually, probably not, that extends IListOptions which extends IDataOptions which implements:

    interface IDataOptions {
        expand?: Array<string>;
        include?: Array<string>;
    }

And those are not valid for customers.list. Thoughts?

Thoughts?

Reply:
If you still prefer to extend IListOptionCreated, please let me know and I will go ahead with the change.

Reply:
Extending `IListOptionsCreated` adds the correct behaviour, minimizing duplication. `expand` "is available on all API requests", see [Expanding Objects](https://stripe.com/docs/api/node#expanding_objects).

And I am unsure about `include`, it was added in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/8907. @CodeAnimal, do you recall where `include` in `IDataOptions` comes from?
https://github.com/tzarger/DefinitelyTyped/blob/5f1e463307cde6379862ca491d36f58c9b04d986/types/stripe/index.d.ts#L7391
This may be cruft that should be removed.